src/mono/wasm/runtime/es6/dotnet.es6.pre.js

```diff
@@ -1,3 +1,5 @@
 if (_nativeModuleLoaded) throw new Error("Native module already loaded");
 _nativeModuleLoaded = true;
-createDotnetRuntime = Module = createDotnetRuntime(Module);
\ No newline at end of file
+Module["getMemory"] = function () { return wasmMemory; }
```

Comment:
I would prefer accessing it via `Module.asm.memory`, we already have it here:
https://github.com/dotnet/runtime/blob/4964d542fccea3e80777a62966a7615d98bbb2ea/src/mono/wasm/runtime/startup.ts#L493

Could you please refactor it and move it to memory.ts ?

Reply:
We can't use Module.asm.memory or Module.wasmMemory. See https://github.com/emscripten-core/emscripten/issues/19832

Reply:
Do you mean that the line 493 above doesn't work ?

Reply:
That's correct, in my testing Module.wasmMemory is undefined and Module.asm.memory is the wrong type to import into another module

Reply:
I never tested snapshot in MT. Could you please fix this place to use your new method ? 
Also, could you please add link from `dotnet.es6.pre.js` to https://github.com/emscripten-core/emscripten/pull/19816 so that we could simplify it when we update emsdk.

Reply:
thanks a lot!